src/rsa/rsa.rs

```diff
@@ -85,7 +97,14 @@ rsa_pkcs1_padding!(RSA_PKCS1_SHA512, &digest::SHA512,
                    &SHA512_PKCS1_DIGESTINFO_PREFIX,
                    "Signing using RSA with PKCS#1 1.5 padding and SHA-512.");
 
-macro_rules! rsa_pkcs1 {
+rsa_pss_padding!(RSA_PSS_SHA256, &digest::SHA256,
+                   "Signing using RSA with PSS padding and SHA-256.");
+rsa_pss_padding!(RSA_PSS_SHA384, &digest::SHA384,
+                   "Signing using RSA with PSS padding and SHA-384.");
+rsa_pss_padding!(RSA_PSS_SHA512, &digest::SHA512,
+                   "Signing using RSA with PSS padding and SHA-512.");
+
+macro_rules! rsa_impl {
```

Comment:
More from the Small Commit Propaganda Committee: I would split the rename of the `rsa_pkcs1` macro out into a separate commit.